frontend/src/pages/about.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useState,  CSSProperties } from 'react';
+import brandonImage from '../assets/BArsenault.jpg';
+
+
+
+const AboutPage: React.FC = () => {
+    const [boxShadow, setBoxShadow] = useState<string>('none');
+    const [boxPosition, setBoxPosition] = useState<string>('translateX(0) scale(1)');
+
```

Comment:
we should add a comment that this needs to be refactored to a separate CSS file w/ the task number once we have a board set up

Reply:
I went ahead and took care of this!